projects/alembic/Dockerfile

```diff
@@ -27,3 +27,4 @@ RUN git clone -b v2.4.2 --depth 1 https://github.com/AcademySoftwareFoundation/o
 
 COPY build.sh *.h *.cc $SRC/
 WORKDIR $WORK/
+ENV OLD_LLVMPASS 1
```

Comment:
Ooo it would be great if a comment could explain what this is.

Reply:
Done.

Reply:
Comments that explain why are better than "what". I think it's clear from the code that something to do with LLVM's passes is being set, the comment would be better if it explained why this is needed.

Reply:
+1. Let's say this is for introspector + link to the relevant bug.

Reply:
How about now?

Reply:
LGTM